VTEX - Marketplace APIs.json

```diff
@@ -3424,7 +3424,164 @@
+                    "required": true,
+                    "style": "form",
+                    "explode": true,
+                    "schema": {
+                      "type": "string",
+                      "default": "{{accountName}}"
+                    }
+                  }
+                ],
+                "responses": {
+                    "200": {
+                      "description": "OK",
+                      "content": {
+                        "application/json": {
+                          "schema": {
+                            "type": "array",
+                            "items": {
+                              "type": "object",
+                              "example":
+                                {
+                                  "account": "grocery1",
+                                  "id": "digibee.shopee",
+                                  "affiliateId": "SHP",
+                                  "salesChannel": "1",
+                                  "channelName": "shopee",
+                                  "channelLogo": "https://s3.amazonaws.com/Marketplace-Integration/Bridge/logos/shopee.png"
+                                },
+                                "properties": {
+                                    "account": {
+                                      "type": "string",
+                                      "description": "Seller's account name."
+                                    },
+                                    "id": {
+                                      "type": "string",
+                                      "description": "Name created by the seller to identify the marketplace."
+                                    },
+                                    "affiliateId": {
+                                      "type": "string",
+                                      "description": "Corresponds to the 3-digit [affiliate](https://help.vtex.com/en/tutorial/configuring-affiliates--tutorials_187) identification code created by the seller."
+                                    },
+                                    "salesChannel": {
+                                      "type": "string",
+                                      "description": "Sales channel (or [trade policy](https://help.vtex.com/en/tutorial/como-funciona-uma-politica-comercial--6Xef8PZiFm40kg2STrMkMV#master-data)) used in the integration."
+                                    },
+                                    "channelName": {
+                                      "type": "string",
+                                      "description": "Name of the marketplace the feed is about."
+                                    },
+                                    "channelLogo": {
+                                      "type": "string",
+                                      "description": "Graphic symbol that identifies the marketplace."
```

Comment:
URL redirecting to the image file of the graphic symbol that identifies the marketplace*